ws-nextjs-app/pages/[service]/new_live/[id]/[[...variant]].page.tsx

```diff
@@ -48,10 +48,8 @@ const getPageData = async ({
 
   const env = getEnvironment(pathname);
   const optHeaders = { 'ctx-service-env': env };
-  const isLocal = !env || env === 'local';
-  const certsNeeded = !isLocal && process.env.INTEGRATION_TEST_BUILD !== 'true';
 
-  const agent = certsNeeded ? await getAgent() : null;
+  const agent = certsRequired(pathname) ? await getAgent() : null;
```

Comment:
Probably worth us, (Content team) to have a ticket to get the reusable code to work with the live page

Reply:
Yeah, Aaron and I had a bit of a chat about this on Slack. I couldn't get the integration tests working, despite it working locally 😭. We also noticed that the integration tests for the live page only check for the heading & nav bar, so even if the page returned a 404, the integration tests would still pass. If you do pick up the ticket, you should be able to cherry-pick the commits from this PR (or maybe you might want to just start again! `git cherry-pick cf8c43f25574444a302bccb1e98fd66cc7249b27`)